tools/run_system_tests.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+
+# Leka - LekaOS
+# Copyright 2020 APF France handicap
+# SPDX-License-Identifier: Apache-2.0
+
+from colorama import Fore, Style
+from time import sleep
+import argparse
+import glob
+import re
+import serial
+import serial.tools.list_ports
+import subprocess
+import sys
+
+
+#
+# MARK: - argparse
+#
+
+
+OS_BIN_FILE_PATH = "_build/LEKA_V1_2_DEV/app/os/LekaOS.bin"
+
+parser = argparse.ArgumentParser(description='Run functional tests')
+
+parser.add_argument('-p', '--port', metavar='PORT',
+                    default='/dev/tty.usbmodem*',
+                    help='serial port path used for the robot')
+
+parser.add_argument('--response-timeout', metavar='RESPONSE_TIMEOUT',
+                    default=30.0,
+                    help='response timeout is seconds')
+
+parser.add_argument('--no-flash-erase', action='store_false',
+                    help='disable flash erase')
+
+parser.add_argument('-d', '--duration', metavar='DURATION',
+                    default=180,
+                    help='duration in seconds to wait for the system to sleep')
+
+parser.add_argument('-s', '--deep-sleep-percentage', metavar='DEEP_SLEEP_PERCENTAGE',
+                    default=95,
+                    help='deep sleep percentage')
```

Comment:
Since both values are related, they should be coherent in these tests. It can be updated on the CI file to update both duration and percentage.

```suggestion
parser.add_argument('-d', '--duration', metavar='DURATION',
                    default=18000,
                    help='duration in seconds to wait for the system to sleep')

parser.add_argument('-s', '--deep-sleep-percentage', metavar='DEEP_SLEEP_PERCENTAGE',
                    default=95,
                    help='deep sleep percentage')
```